man/truelength.Rd

```diff
@@ -34,6 +38,12 @@ alloc.col(DT,
     (perhaps in your .Rprofile); e.g., \code{options(datatable.alloccol=10000L)}.
 
     Please note: over-allocation of the column pointer vector is not for efficiency \emph{per se}; it is so that \code{:=} can add columns by reference without a shallow copy.
+
+    \code{setallocrow} is a utility function that prepares columns for fast row operations (delete or insert (not implemented yet)) by reference and manages row capacity.
+    Before deleting or inserting rows by reference, columns must be resizable.
+    \code{setallocrow} ensures all columns are in the appropriate state by converting ALTREP columns to materialized form and reallocating
```

Comment:
I think this needs a bit of clarification -- when I read it, my thought was "oh, we can't do `.ROW:=NULL` on a table with `ALTREP` columns unless we first `setallocrow()`", but I don't see any examples/tests to that effect.

OTOH, I think (?) we strive to expand all `ALTREP` columns, so I haven't been able to come up with such an example